package.json

```diff
@@ -76,7 +76,7 @@
     "gulp-uglify-es": "^3.0.0",
     "ini": ">=3.0.1",
     "kind-of": ">=6.0.3",
-    "lit-analyzer": "^1.2.1",
+    "lit-analyzer": "^0.0.6",
```

Comment:
Is this the right thing to do?  I ran the suggested command to remove 3 vulnerabilities, which might have resulted in this change of the lit-analyzer version.  But version 0.0.6 is older than 1.2.1, right?

Reply:
@jcscottiii I recall you saying that there was something more complex about upgrading lit-analyzer to a new version that was still in beta.  Dan made this change and it seems to pass tests.  Is this change OK (even if we need to do more later)?

Reply:
The change involves moving from this 3rd party lit analyzer to a [1st party lit analyzer](https://www.npmjs.com/package/@lit-labs/analyzer). But before adopting that one, we should wait until it is stable (at least 1.0.0).

It's safe to make the change. But I'm more afraid that we may be missing out on more lint rules by downgrading. Here's the [CHANGELOG](https://github.com/runem/lit-analyzer/blob/master/CHANGELOG.md) over time for the current lit analyzer. Since those vulnerable packages are only for lit analyzer for linting, I think it safe to keep it at 1.2.1.

Reply:
Thanks for the reminder.  Let's not downgrade. Let's keep it at 1.2.1.

Reply:
Restored to 1.2.1.